### PR TITLE
Namespace changes to testing-and-debugging.md

### PR DESCRIPTION
**Pull request by MikeDodaro**

### CODE REVIEW

**Comment by MikeDodaro** [comment]
Links will have to change eventually:
@"microsoft.quantum.primitive.message"
@"microsoft.quantum.primitive.assert"
@"microsoft.quantum.primitive.assertprob"

File: articles/techniques/testing-and-debugging.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ This makes functions returning `()` a useful tool for embedding assertions and d
 
 ### Logging
 
-The primitive function @"microsoft.quantum.primitive.message" has type `String -> ()` and enables the creation of diagnostic messages.
+The Intrinsic function @"microsoft.quantum.primitive.message" has type `String -> ()` and enables the creation of diagnostic messages.
```
Comment (cgranade) [inline articles/techniques/testing-and-debugging.md]:
```suggestion
The intrinsic function @"microsoft.quantum.intrinsic.message" has type `String -> ()` and enables the creation of diagnostic messages.
```

File: articles/techniques/testing-and-debugging.md [hunk 2]
```diff
@@ -264,7 +264,7 @@ Notice how the ids of the qubits are shown at the top in their significant order
 ```qsharp
 namespace Samples
 {
-    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Intrinsic;
     open Microsoft.Quantum.Extensions.Diagnostics;
```
Comment (cgranade) [inline articles/techniques/testing-and-debugging.md]: The "Extensions" part of the namespace name has been removed for simplicity.
```suggestion
    open Microsoft.Quantum.Diagnostics;
```

File: articles/techniques/testing-and-debugging.md [hunk 3]
```diff
@@ -321,7 +321,7 @@ The following example shows you how you can use both @"microsoft.quantum.extensi
 ```qsharp
 namespace app
 {
-    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Intrinsic;
     open Microsoft.Quantum.Extensions.Diagnostics;
```
Comment (cgranade) [inline articles/techniques/testing-and-debugging.md]:
```suggestion
    open Microsoft.Quantum.Diagnostics;
```

**Pull request merged**